website/docs/docs/guides/migration-guide/upgrading-to-0-18-0.md

```diff
@@ -51,3 +53,4 @@ Please be aware of the following changes in v0.18.0. While breaking, we do not e
 - [Redshift profile](redshift-profile#specifying-an-iam-profile)
 - [`asset-paths` config](asset-paths) (also updated [dbt_project.yml](dbt_project.yml.md) and the [description](description) docs to match)
 - [`impersonate_service_account` in the BigQuery profile configuration](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#service-account-impersonation)
+- [Deferring to previous run state](run#deferring-to-previous-run-state)
```

Comment:
this links to the wrong place because of the logic (read: bug) in the link checker component. Can you make sure this branch incorporates the latest `current` commit? Would like to make sure that https://github.com/fishtown-analytics/docs.getdbt.com/pull/334 works correctly now :)

Reply:
Confirmed that this works locally when I pulled in the change from `current`